api_server/auth/utils/change_password_tokens/email_letters.py

```diff
@@ -0,0 +1,57 @@
+import html
+import re
+
+from celery.app.utils import Settings
+from jinja2 import Template
+
+from auth.models import ChangePasswordToken
+from common.constants.auth import ChangePasswordLetterConstants
+
+
+class ChangePasswordLetter:
+
+    def __init__(self, db_token: ChangePasswordToken, server_config: Settings) -> None:
+        self.db_token = db_token
+        self._server_config = server_config
+
+    @property
+    def change_password_url(self):
```

Comment:
According to Python's getter - setter philosophy, `@property` should be named as an attribute. Is it better to rename it something like `password_url`, `changed_password_url`?

As a proof, I can add link with example of working with properties: [link](https://realpython.com/python-property/#:~:text=In%20this%20code%20snippet%2C%20you%20create%20Circle.)